impl/src/main/java/com/okta/sdk/impl/config/ClientConfiguration.java

```diff
@@ -167,6 +205,10 @@ public String toString() {
                 ", cacheManagerTti=" + cacheManagerTti +
                 ", cacheManagerCaches=" + cacheManagerCaches +
                 ", baseUrl='" + getBaseUrl() + '\'' +
+                ", authorizationMode='" + getAuthorizationMode() +
+                ", clientId='" + getClientId() +
+                ", scopes='" + getScopes() +
+                ", privateKey=" + getPrivateKey() +
```

Comment:
this could be secret, if we support armored strings, you do NOT want to print it.
The ClientConfiguration class could have two properties one for File, and one for the parsed string, logging the file name isn't an issue, but the PEM string would be